README.md

```diff
@@ -68,15 +70,15 @@ Open the zip that you downloaded from one of the links above, and drag the A32NX
 See below for the location of your Community folder:
 
 For the Microsoft Store edition AND/OR Gamepass edition:
-- Copy the "A32NX" folder into your community package folder. It is located in:
+- Copy the "A32NX" folder into your community package folder. The folder is located in:

```

Comment:
This change is completely unnecessary. If the change is intended to resolve ambiguity between the A32NX folder and the community package folder, it fails in doing so, because "the folder" is just as ambiguous as "it".